scripts/phase_gates/b0_4_gate.py

```
@@ -40,6 +55,46 @@ def main() -> int:
         )
+        run(
+            ["python", "scripts/security/phase4_enforcement_probe.py"],
+            "b0_4_phase4_security_probe.log",
+            env=env,
+        )
+
+        # EG4.7: run unchanged EG3.4 customer-profile perf suite under RLS-enabled schema.
+        base_url = env.get("R3_API_BASE_URL", "http://127.0.0.1:8000")
+        uvicorn_log = EVIDENCE_DIR / "b0_4_r3_uvicorn.log"
+        with uvicorn_log.open("w", encoding="utf-8") as fh:
+            server = subprocess.Popen(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)